doc/nvim-treesitter.txt

```diff
@@ -527,22 +528,22 @@ in Lua.
 								*hl-TSKeyword*
 `TSKeyword`
 For keywords that don't fall in previous categories.
-`keyword`
+Capture: `keyword`
```

Comment:
I like this.

Ideally, we should have a lua file somewhere with all of this, that we use to generate the definitions of the highlight groups, this docs, and check in the CI.

Reply:
Isn't there somewhere a file where the highlight groups get assigned to captures

Reply:
Yes, it is done in `plugin/nvim-treesitter.vim`

Reply:
in `lua/nvim-treesitter/highlight.lua`